docs/troubleshoot/troubleshoot-apiml-error-codes.md

```diff
@@ -641,7 +677,7 @@ The following error message codes may appear on logs or API responses. Use the f
 
   **Reason:**
 
-  The Zaas Client configuration does not contain the path to the key store.
+  The Zaas Client configuration didn't contain the path to the key store
```

Comment:
Keep verb in the present tense and avoid contractions. "The Zaas Client configuration does not contain the path to the trust store" is correct.